docs/wsgi-middleware.rst

```diff
@@ -1,66 +1,62 @@
 
 
-Using as Middleware
--------------------
+Using the Middleware
+--------------------
 
-To add GraphQL API to your project using ``GraphQLMiddleware``, instantiate it with your existing WSGI application as a first argument, type defs as second and resolvers as third::
+To add GraphQL API to your project using ``GraphQLMiddleware``, instantiate it with your existing WSGI application as a first argument and your schema as the second::
 
     # in wsgi.py
     import os
 
     from django.core.wsgi import get_wsgi_application
-    from ariadne import GraphQLMiddleware
+    from ariadne import GraphQLMiddleware, make_executable_schema
     from mygraphql import type_defs, resolvers
 
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "mydjangoproject.settings")
 
+    schema = make_executable_schema(type_defs, resolvers)
     django_application = get_wsgi_application()
-    application = GraphQLMiddleware(django_application, type_defs, resolvers)
+    application = GraphQLMiddleware(django_application, schema)
 
-Now direct your WSGI container to `wsgi.application`. GraphQL API is available on ``/graphql/`` by default, but this can be customized by passing path as a fourth argument::
+Now direct your WSGI container to `wsgi.application`. GraphQL API is available on ``/graphql/`` by default, but this can be customized by passing path as the third argument::
 
     # GraphQL will now be available on "/graphql-v2/" path
-    application = GraphQLMiddleware(
-        django_application, type_defs, resolvers, "/graphql-v2/"
-    )
+    application = GraphQLMiddleware(django_application, schema, "/graphql-v2/")
 
 
 Customizing context or root
 ---------------------------
 
-:py:class:`GraphQLMiddleware` defines two methods that you can redefine in inheriting classes:
+:py:class:`GraphQL` defines two methods that you can redefine in inheriting classes:
 
-.. method:: GraphQLMiddleware.get_query_root(environ, request_data)
+.. method:: GraphQL.get_query_root(environ, request_data)
 
     :param environ: `dict` representing HTTP request received by WSGI server.
     :param request_data: json that was sent as request body and deserialized to `dict`.
     :return: value that should be passed to root resolvers as first argument.
 
-.. method:: GraphQLMiddleware.get_query_context(environ, request_data)
+.. method:: GraphQL.get_query_context(environ, request_data)
 
     :param environ: `dict` representing HTTP request received by WSGI server.
     :param request_data: json that was sent as request body and deserialized to `dict`.
     :return: value that should be passed to resolvers as ``context`` attribute on ``info`` argument.
 
-The following example shows custom GraphQL middleware that defines its own root and context::
+The following example shows custom a GraphQL server that defines its own root and context::
```

Comment:
I would also add a paragraph with code example to this section telling how to use this custom `MyGraphQL` with `GraphQLMiddleware` introduced ealier.